### ISSUE TITLE
aria-roledescription for child elements?

### ISSUE THREAD

**Issue by JAWS-test** [w3c/aria#1372]
With aria-roledescription I can specify a different role description. This is already problematic with simple widgets, because then the actual role is not perceivable with the screen reader and thus the operation is unclear (see https://github.com/w3c/aria/issues/1132).

But I wonder to what extent aria-roledescription is useful for complex widgets or nested roles and what output is expected by AT? For many nested roles (like table, row and cell; listbox and option), the screen reader outputs only the outer role (table, listbox), but not the role for all child elements (cell, option). If now the child elements have aria-roledescription: Should this also be ignored by the screen reader or should it always be output (e.g. for each table cell or each list entry of a listbox). I don't think both is optimal and tend to forbid aria-roledescription on roles like row, cell, option.

**Comment by stes-acc** [comment]
Results of deep dive on the topic:

1.	It is helpful  to mention base role in roledescription when using roledescription on widgets. The ARIA spec https://w3c.github.io/aria/#aria-roledescription chapter can be improved with a sentence explaining this.

2.	Any text not mentioning base role when using roledescription on widgets is problematic since it masks base role and is therefore considered as authoring error. Bad practices get no special mentioning neither in the ARIA specification nor the ARIA APG.

3.	It is expected that for overrides of base role of region containers etc. using roledescription on widgets, the roledescription is spoken instead the base role when focusable descendants are focused. Observed violations can be filed to the AT vendor and are expected to be acknowledged and fixed.

4.	The reason for overriding base role speech output with roledescription content was decided on group consensus when the property has been introduced to AT as result of feedback of blind users that were confused by double-speak of roledescription and base role. 

5.	The group has created a respective special issue [https://github.com/w3c/aria/issues/1480)] where all further discussion should be continued.

6.	Update with this info:

[18:08] <StefanS_> [https://github.com/w3c/aria/issues/1373]
[18:08] <StefanS_> [https://github.com/w3c/aria/issues/1132]
[18:09] <StefanS_> [https://github.com/w3c/aria/issues/1372]
